import os
import sys
from CeleryPy import log
from DB import DB
import Capture 
import Image
x=DB.getcoordinates().__str__()
log('ya', message_type='error', title='FUNCO')

